### PR TITLE
Change color of send status indicator

### LINKED ISSUE #376: Background color of compose status in night theme is white.
The background and text color of compose status in chatbox is white under night theme.

screenshot_20170210-005751

### PR DESCRIPTION
**Pull request by nitish1211**
Fixes #376

The following colours now show in the sending status indicator.

**Night theme:**
screenshot_20170210-035155

**Day Theme:**

screenshot_20170210-035259

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@nitish1211, it looks like you've already signed the Dropbox CLA. Thanks!

**Comment by Sam1301** [comment]
Thanks for working on this :+1: , just a UI preference, I think it would be better if we could match the background color of compose status to the chatbox color in both themes.

**Comment by nitish1211** [comment]
@Sam1301 Done.

**Comment by Sam1301** [comment]
Nice, works as expected in day theme. For night theme, I think the background color of the compose status could be made to match the color of the chatbox since we are doing _the same in the day theme_

current
screenshot_20170213-050329

expected
screenshot_20170213-052246

**Comment by kunall17** [comment]
This seems much better and required, also how about changing the chatBox text colors and the timestamp color, maybe 2 shades lighter?

**Comment by kunall17** [comment]
The text color seems fines, I was judging from the screenshots you posted but forget that they are while we are sending messages!

Btw can you fix these two bugs in different commits in this PR itself?

- Different icon colors in the chat box
- In the stream textView in the message header that triangle thing should be coloured dark!

screen shot 2017-02-14 at 11 27 31 pm
screen shot 2017-02-14 at 11 28 53 pm

**Comment by nitish1211** [comment]
@kunall17 Is there a specific reason why some features are excluded from the night theme?
In these cases:
Prevents setting of the color of the arrow : [Line 357 ](https://github.com/zulip/zulip-android/blob/master/app/src/main/java/com/zulip/android/activities/RecyclerMessageAdapter.java#L357)

Setting of the color bar: [Line 409](https://github.com/zulip/zulip-android/blob/master/app/src/main/java/com/zulip/android/activities/RecyclerMessageAdapter.java#L409)

**Comment by kunall17** [comment]
>  why some features are excluded from the night theme?

Didn't understand you correctly?

**Comment by nitish1211** [comment]
`if (!isCurrentThemeNight)` [here](https://github.com/zulip/zulip-android/blob/master/app/src/main/java/com/zulip/android/activities/RecyclerMessageAdapter.java#L357)  why has this condition been used before setting color of the arrow(triangular thing) and when setting the color of the color bar( which is the  colored bar you would notice enclosing the messages under a particular stream on the left edge of the screen).

I was just wondering if there was any design specific reason why this was done.

**Comment by nitish1211** [comment]
Here I've made changes to the Night theme to make it slightly similar to the day theme.
1)The leftBar is now visible
2)The arrow has the same color as the label.

Before:
screenshot_20170216-014345


After:
screenshot_20170216-015830

File: app/src/main/res/layout/chatbox.xml [hunk 1]
```diff
@@ -76,25 +77,24 @@
...
 
     <LinearLayout
         android:id="@+id/composeStatus"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:orientation="horizontal"
         android:layout_gravity="center"
         android:background="@color/colorPrimary"
         android:gravity="center|center_vertical"
+        android:orientation="horizontal"
```
Comment (Sam1301) [inline app/src/main/res/layout/chatbox.xml]: nit: only ordering of attributes changed

File: app/src/main/res/layout/chatbox.xml [hunk 2]
```diff
@@ -76,25 +77,24 @@
...
         android:visibility="gone">
 
         <ProgressBar
-            android:layout_width="15dp"
-            android:layout_height="15dp"
             style="@style/AppTheme.DayNight"
-            />
+            android:layout_width="15dp"
+            android:layout_height="15dp" />
```
Comment (Sam1301) [inline app/src/main/res/layout/chatbox.xml]: same as above

**Comment by kunall17** [comment]
Also it would be better if you split up the fixing of message header colors!

File: app/src/main/res/layout/chatbox.xml (hunk 1 above)
Comment (nitish1211) [inline app/src/main/res/layout/chatbox.xml]: This happens on **formatting** the code. Android studio sorts the attributes in a particular order.

Comment (Sam1301) [inline app/src/main/res/layout/chatbox.xml]: ohk.. ya right :+1: , its fine then :)

**Comment by nitish1211** [comment]
@kunall17 Done. Added a commit for color in the header and side bar in the night theme.

**Comment by kunall17** [comment]
These left color bars were not added on purpose, minimal colors are to be included in the dark theme, remove them! and it's better to have the old format (using isCurrentThemeNight)

**Comment by nitish1211** [comment]
So you prefer color in header arrows but not in sidebar or should it be left as it was earlier? @kunall17 

Like this :
screenshot_20170215-210738

**Comment by kunall17** [comment]
Yeah!

**Comment by nitish1211** [comment]
@kunall17 Done. Added the changes you requested. :+1:

**Comment by kunall17** [comment]
Looks nice, but the color tint on the person icon is not the same color with the other colors! 

screen shot 2017-02-22 at 9 38 23 pm

**Comment by kunall17** [comment]
Also on the day theme this result is being seen! 
screen shot 2017-02-22 at 10 02 22 pm

**Comment by nitish1211** [comment]
@kunall17 This is happening due to the difference in color between the icon's default color which is not dark enough. 

This is the person icon:

ic_action_person

These are the other icons:

ic_photo_camera_black_24dp

ic_send_24dp


Which are darker by default causing them to appear darker. To change the darkness there are two options either to change the icon file or add another color to the person icon which is slightly darker so that both of them look the same. Which one would you prefer?

**Comment by nitish1211** [comment]
Btw this is how it looks on making all icons the same by making the others lighter

screenshot_20170223-210605

screenshot_20170223-210613

Is this fine?

**Comment by kunall17** [comment]
No in the light theme it should be black icons only!

**Comment by nitish1211** [comment]
screenshot_20170223-231607

screenshot_20170223-231559

How about this?

**Comment by kunall17** [comment]
Merged, thanks @nitish1211 👍

**Pull request closed without merging**